src/pages/settings/Profile/CustomStatus/SetTimePage.tsx

```diff
@@ -40,6 +43,7 @@ function SetTimePage({translate, customStatus}) {
             />
             <View style={styles.flex1}>
                 <TimePicker
+                    // @ts-expect-error TODO: remove this comment once TimePicker (https://github.com/Expensify/App/issues/36129) is migrated to TS
```

Comment:
[TimePicker](https://github.com/Expensify/App/pull/37422) was just merged, merge the newest main and remove this comment